src/lang/scheme/r7rs.ss

```diff
@@ -102,7 +114,7 @@ package: scheme
 
   (syntax-case stx ()
     ((_ (id ids ...) decl ...)
-     (identifier-list? #'(id ids ...))
+     (andmap identifier-or-number? #'(id ids ...))
```

Comment:
Why not `stx-andmap`? Because `identifier-or-number?` already destructures?

Reply:
The `'#(id ids ...)` construct creates an uwraped list, so it can be iterated directly.